src/i18n/en.js

```diff
@@ -56,6 +56,14 @@ const messages_en = {
     volume_500: 'Between 500k and 1m',
     volume_do_not_know: `I don't know`,
   },
+  big_query: {
+    free_btn_redirect: `Conoce dsfhdsfbdfbsdf`,
+    free_text_strong: `¡Potencia tu fgvseghdsfhb de BigQuery!`,
+    free_text_summary: `Integra Google cxxxxxx reportes detallados sobre tus campañas,tus suscriptores y otros datos de tu cuenta en un tablero personalizable.`,
+    free_title: `Big Query`,
+    free_ul_item_one: `Obtén valiosos xcxcxcx cuenta`,
+    free_ul_item_two: `Prodas filtrar los aaascsd tus campañas o seleccionar solo algunas en particular.`,
+  },
```

Comment:
¿@cbernat cómo hacen normalmente para que los contenidos no sean un show-stopper?

Reply:
Puede salir una versión sin contenidos sin usar intl o intentar que los contenidos sean creíbles, dependiendo la funcionalidad.

Reply:
Para contenidos en inglés lo traducimos lo mejor que podemos o ponemos el mismo contenido que esta en español.

Reply:
Le propuse a @kapoc3 escribirlos tanto en Inglés como en Español, compartirlos con Juliana para tener las correcciones mínimas para que sean aceptables (en todo caso en un futuro puede volver con más mejoras), aplicarlas y continuar.

Reply:
in this ocation i translated the messages for labels

Reply:
👍 resolving the conversation...